examples/advanced_behaviors/set_value/index.xml.njk

```diff
@@ -75,6 +75,10 @@ tags: advanced_behaviors
         shadowOffsetY="-5"
         shadowOpacity="0.2"
         shadowRadius="5"/>
+      <style
+        id="PickerModal__overlay"
+        backgroundColor="#1f1f1fa0"
```

Comment:
Hard to tell from the syntax. Are these values deleted so they can be provided further down stream? Are they required fields?

Reply:
there's not deleted, I think the GH syntax coloring makes it looks like it's removed but it's not.